components/scream/src/physics/p3/tests/p3_run_and_cmp.cpp

```diff
@@ -219,9 +219,6 @@ struct Baseline {
       const auto& f = fdi.getfield(i);
       ekat::write(&f.dim, 1, fid);
       ekat::write(f.extent, f.dim, fid);
-#ifdef SCREAM_FORCE_RUN_DIFF
```

Comment:
Why is this no longer needed? What's forcing the run diff now?

Reply:
Because we want the difference to be "authentic", IE coming from the real p3 code. This is why the code was introduced to p3_rain_sed_impl above.